drake_cmake_installed/src/find_resource/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Simply tests the Drake find resource api from outside of Drake (i.e. finds and loads resources that are part of the Drake install).
+Simply tests the Drake find resource API from outside of Drake i.e., finds and loads resources that are part of the Drake install.
```

Comment:
Missing comma before `i.e.,`

Reply:
Now, see this is what comes of enforcing grammar rules that aren't even conformed to within the boundary of the United States to British speaking people. 

:all_the_things:

Reply:
I think I will stop using `i.e.` in protest :)

Reply:
Comma after i.e. is pretty well enforced in American English and pretty much alien to British English.

Reply:
I never consumed that much American content, but it was still non-negligible and was surprised to discover this syntax on arriving in Cambridge. There are many web links which note that this is not universally used in the US as well.

I would rather we focus on development than fine-grained US/British differences such as this. Both methods are clear and do not disrupt the flow of the comments to be a consistency issue.